packages/server/src/utils/workflow.ts

```diff
@@ -158,6 +162,8 @@ function appendEventDataToAnnotations(
     downloadLlamaCloudFilesFromNodes(sourceNodes); // download files in background
   }
 
+  // TODO: trigger generate component code and save to componentsDir
```

Comment:
this will add a lot of latency, we will trigger generating the component code in a own function in the generate.ts script